tests/ci/integration/run_openvpn_integration.sh

```diff
@@ -52,10 +58,21 @@ function openvpn_build() {
 
 # TODO: Remove this when we make an upstream contribution.
 function openvpn_patch_build() {
-  for patchfile in $(find -L "${OPENVPN_PATCH_BUILD_FOLDER}" -type f -name '*.patch'); do
-    echo "Apply patch $patchfile..."
-    patch -p1 --quiet -i "$patchfile"
-  done
+  case "$BRANCH_NAME" in
+    "release/2.6")
+      patchfile="${OPENVPN_PATCH_BUILD_FOLDER}/aws-lc-openvpn2-6-x.patch"
```

Comment:
NP: Very minor, but is there a way we could just reconfigure the `release/2.6` string to match our patch directory/file naming scheme? Maybe something like taking the `/` out and directing it to a `release2.6` directory or file.

Reply:
I think this should be fine for now, should we decide to add another branch in the future happy to refactor the code as necessary. Currently, not sure what the approach would be - probably some string manipulation stuff that would be overly complex for the current state of this script.